src/mudblazor/MudBlazor.Template/Program.cs

```diff
@@ -68,7 +69,7 @@
 #endif
 builder.Services.AddDatabaseDeveloperPageExceptionFilter();
 
-builder.Services.AddIdentityCore<ApplicationUser>(options => options.SignIn.RequireConfirmedAccount = true)
+builder.Services.AddIdentityCore<ApplicationUser>(options => options.SignIn.RequireConfirmedAccount = false)
```

Comment:
This change fixes the "Error: Invalid login attempt." message that users see when they try to log in after registering.  What's happening is that the site is requiring confirmation, but at the same time has no email provider configured.  Thus, users would have to manually connect to the database, update their record, then sign in.  Quite a pain.  With this as `false`, users are immediately signed in upon registering.  My only concern is that it's less secure.

Reply:
You don't need to connect to the database manually to confirm, there is a **RegisterConfirmation** page that provides a "Click here to confirm your account" link. After registration, the user should be directed to that page -> `Components/Account/Pages/RegisterConfirmation.razor`. You can probably style the link so it's more noticeable though.

Reply:
Ahh ok, thanks for explaining, @Anu6is!  I have changed this back to `true` and will try to improve the "Register Confirmation" link.